AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -213,7 +213,9 @@ - (void)setLogUrl:(NSString *)logUrl {
 }
 
 - (void)setAppSecret:(NSString *)appSecret {
-  self.ingestion.appSecret = appSecret;
+  if([self.ingestion isKindOfClass:[MSAppCenterIngestion class]]) {
```

Comment:
If the type check is here we could merge https://github.com/Microsoft/AppCenter-SDK-Apple/commit/a10fb60af3ee194bd8c8773a51b3508b48b3ba34 a week ago

Reply:
Yep, I agree.